### ISSUE TITLE
Help Needed - Nested ListView

### ISSUE THREAD

**Issue by shabib3** [NativeScript/NativeScript#5935]
Hi guys,

I tried to find any solution about this in other issues but nothing helped me.
I am trying to create something exactly like this -> [video sample](https://www.youtube.com/watch?v=94Hk-Nk-4dw&feature=youtu.be)

As you can see in the video it seems to be a ListView inside ListView (or ListView inside ScrollView).
The page splits to 3 Section - 

- `Cover Photo` - upper section.
- `Menu Bar` - that use for changing the content and notice that it becomes floating when it gets into less than 0 Y Offset.
- `Content` section - bottom section.

At first I thought that it is one ListView, but as you can see in the begging I was able to scroll only the `Content` section without moving the top section (I am not sure 'facebook' wants it to happen).

So to sum up, I want really exactly the same, a scrollable page that contains, an upper section that hold a cover picture, a menu bar that controls that shown content, and a changeable and scrollable content section that changes his content by the menu bar.

Any idea how can I get it?

**Comment by shabib3** [comment]
I just want to share with you what I am thinking about `facebook` implementation of this...
As you can see in the video I found kind of thing that I am not sure they want to happen...The content section can scroll without the all page scroll...([in 0:03 in the video](https://youtu.be/94Hk-Nk-4dw?t=3s))
This is kind of showing that they use ListView inside ListView (Scrollable element inside Scrollable element)...The inner ListView is the changeable content section

Is there any way to implement it in nativescript?

**Comment by NickIliev** [comment]
@shabib3 multiple nested listviews [are not supported](https://github.com/NativeScript/NativeScript/issues/92). In fact, netting a listview within a listview will cause performance issues due to the current implementation where each listview cell is being recycled ( and this would break the recycling and virtualization for cells that are containing a nested listview).

There are separate solutions and number of implementations for creating the different visual effects like parallax scroll or animated header, but there is no option using a scroll view with nested listview (due to event propagation) or nested listviews (event propagation and performance issues).

**Comment by tigrenok00** [comment]
@NickIliev 
Could you please advise on the possible implementation of this with a ListView?
I want to achieve the parallax effect which is shown in this great example, however, it uses a ScrollView and a simple StackLayout, and we need to use it with RadListView inside as well.
https://play.nativescript.org/?template=play-ng&id=DXJkOJ&v=9

@shabib3  
I would love to hear which way you decided to go if you found a solution, thanks.

**Comment by gabrielsmelo** [comment]
@shabib3 
@tigrenok00 

I'm having the same needing. Did you found a solution?
